crates/node/src/sync/blobs.rs

```diff
@@ -65,11 +65,16 @@ impl Node {
             }
         };
 
-        let our_sending_key = self
+        let possible_sending_key = self
             .ctx_manager
-            .get_own_signing_key(&context.id, &our_identity)?;
+            .get_sender_key(&context.id, &our_identity)?;
 
-        let shared_key = SharedKey::new(&our_sending_key, &their_identity);
+        let sending_key = match possible_sending_key {
+            Some(key) => key,
+            None => todo!(),
+        };
+
+        let shared_key = SharedKey::new(&sending_key, &their_identity);
```

Comment:
we are not to use sender keys for sync, but DHKE

Reply:
Not sure what you mean, check if fixed in new commit.

Reply:
the rest are resolved, but here it shouldn't be our "sending key" x their identity, but our private key x their public key.